From 854c7f9559bf1ce6616b61528b3859d4a9ce157a Mon Sep 17 00:00:00 2001
From: Feng Xiao <xf@rock-chips.com>
Date: Wed, 23 Mar 2016 11:13:11 +0800
Subject: [PATCH] clk: rockchip: rk3366: modify cpuclk_rate_table

add 1296MHz, 1104MHz and 216MHz to the cpuclk_rate_table list

Change-Id: I1ea7ee432b7c69b89cb3c11a74e67d9d6af1a5dd
Signed-off-by: Feng Xiao <xf@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3366.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/clk/rockchip/clk-rk3366.c b/drivers/clk/rockchip/clk-rk3366.c
index 51ef4ce37b67..6997403770ac 100644
--- a/drivers/clk/rockchip/clk-rk3366.c
+++ b/drivers/clk/rockchip/clk-rk3366.c
@@ -190,13 +190,16 @@ static struct rockchip_cpuclk_rate_table rk3366_cpuclk_rates[] __initdata = {
 	RK3366_CPUCLK_RATE(1512000000, 1, 5, 5),
 	RK3366_CPUCLK_RATE(1488000000, 1, 4, 4),
 	RK3366_CPUCLK_RATE(1416000000, 1, 4, 4),
+	RK3366_CPUCLK_RATE(1296000000, 1, 4, 4),
 	RK3366_CPUCLK_RATE(1200000000, 1, 3, 3),
+	RK3366_CPUCLK_RATE(1104000000, 1, 3, 3),
 	RK3366_CPUCLK_RATE(1008000000, 1, 3, 3),
 	RK3366_CPUCLK_RATE( 816000000, 1, 2, 2),
 	RK3366_CPUCLK_RATE( 696000000, 1, 2, 2),
 	RK3366_CPUCLK_RATE( 600000000, 1, 1, 1),
 	RK3366_CPUCLK_RATE( 408000000, 1, 1, 1),
 	RK3366_CPUCLK_RATE( 312000000, 1, 1, 1),
+	RK3366_CPUCLK_RATE( 216000000, 1, 1, 1),
 };
 
 static struct rockchip_clk_branch rk3366_i2s_8ch_fracmux __initdata =
-- 
2.35.3

